locales/es/messages.jsonc

```diff
@@ -0,0 +1,1433 @@
+      },
+      "features_RoSealSettingDropdown_title": {
+            "message": "Elemento \"Configuración de RoSeal\" en el menú desplegable de configuración de Roblox"
+      },
+      "features_AccurateHomeContinueSort_title": {
+            "message": "Orden preciso de \"Continuar\" en la página de inicio no curada"
+      },
+      "features_CSSFixes_title": {
+            "message": "Varias correcciones de estilo (CSS) en todo el sitio"
+      },
+      "features_MoveExperienceEvents_title": {
+            "message": "Mover eventos de experiencia a su propia pestaña"
+      },
+      "features_RefreshlessGroupNavigation_title": {
+            "message": "Navegación de grupos sin necesidad de actualizar la página"
+      },
+      "features_RefreshlessGroupNavigation_description": {
+            "message": "Cuando se habilita, navegar a otro grupo desde la organización de grupos/grupos pendientes no requerirá una actualización de página."
+      },
+      "features_ItemAdditionalProductInfo_title": {
+            "message": "Mostrar información adicional del producto"
+      },
+      "features_RefreshAvatarItemDetails_title": {
+            "message": "Botón para actualizar detalles de elementos de avatar"
+      },
+      "features_MarketplaceColorFilters_title": {
+            "message": "Filtros de color del Marketplace"
+      },
+      "features_RefreshAvatarItemDetails_sub_FastFreeBuy_title": {
+            "message": "Compras rápidas y gratuitas de elementos de avatar"
+      },
+
+      "popup_extensionname": {
+            "message": "{sealEmoji} RoSeal"
+      },
+      "popup_actions_robloxgroup": {
+            "message": "Grupo de Roblox"
+      },
+      "popup_actions_settings": {
+            "message": "Configuración"
+      },
+      "popup_actions_website": {
+            "message": "Sitio web"
+      },
+      "popup_permissions": {
+            "message": "Permisos opcionales"
+      },
+      "popup_permissions_cookie": {
+            "message": "Cookies"
+      },
+      "popup_extensiondata_show": {
+            "message": "Mostrar"
+      },
+      "popup_extensiondata_hide": {
+            "message": "Ocultar"
+      },
+      "popup_extensiondata": {
+            "message": "Datos de la extensión"
+      },
+      "popup_extensiondata_download": {
+            "message": "Descargar datos"
+      },
+      "popup_extensiondata_request": {
+            "message": "Solicitar datos"
+      },
+      "popup_extensiondata_import": {
+            "message": "Importar"
+      },
+      "popup_extensiondata_import_alert_succcess": {
+            "message": "Datos importados exitosamente."
+      },
+      "popup_extensiondata_import_alert_error": {
+            "message": "Por favor, inténtalo de nuevo con un archivo de datos válido."
+      },
+
+      "buymeacoffee": {
+            "message": "{coffeeEmoji} Comprame un café"
+      }
+}
+
+// Translated with love by Furrycality UwU - English to Spanish
```

Comment:
due to how syncing between locales work, the credit comment will be erased when i run the workflow

im not sure how to properly credit at the moment

Reply:
dont worry

Reply:
you can add a translator role in the roseal discord server, its an a idea, or make a translators credits in the credit section